docs/comparison.md

```diff
@@ -24,7 +24,7 @@ a headless raspbian image is also maintained by the community
 | ReSpeaker                                                                                                  |             2-mic<br>4-mic squared<br>4-mic linear<br>6-mic             |  2-mic<br>4-mic squared<br>4-mic linear<br>6-mic  |                       manual install                         |                  
 | USB                                                                                                        |                                   Yes                                   |                        Yes                        |                       Yes                         |                    
 | SJ-201                                                                                                     |                                   Yes                                   |                        Yes                        |                       **WIP**                         |                   
-| Google AIY v1                                                                                              |                      Yes<br>*manual configuration*                      |           Yes<br>*manual installation*            |      manual install      |                     
+| Google AIY v1                                                                                              |                      Yes<br>*manual configuration*                      |           Yes<br>*manual installation*            |      [manual install](https://github.com/andlo/picroft-google-aiy-voicekit-skill/blob/master/install_AIY.sh)      |                     
```

Comment:
@j1nx can you review this, i think this script is doing much more than is needed

perhaps add a new helper script directly to ovos-raspbian repo

Reply:
I am on the process of that now. This does way more, and some is not applicable

Reply:
Line 13 to 21 needs checking, what those firmware package does exactly. Then migrated to ovos-i2csound detection and configuration system.

The rest is all simple stuff that can be done already within that system. (This answes my own question at the other PR)